### PR TITLE
Add extra config output

### PR DESCRIPTION
**Pull request by yencli**
Add a list of private data contents (String) in pyslurm configuration output with the key name as "private_data_list".

### CODE REVIEW

File: pyslurm/pyslurm.pyx [hunk 1]
```diff
@@ -333,6 +333,21 @@ cpdef dict slurm_load_slurmd_status():
 # Slurm Config Class
 #
 
+def get_private_data_list(data):
+    u"""Return the list of enciphered Private Data configuration.
```
Comment (giovtorres) [inline pyslurm/pyslurm.pyx]: Missing closing triple quote

File: pyslurm/pyslurm.pyx [hunk 2]
```diff
@@ -333,6 +333,21 @@ cpdef dict slurm_load_slurmd_status():
...
+    
+    result = []
+    exponent = 7
+    types = ['jobs', 'node', 'partitions', 'usage', 'users', 'accounts', 'reservations', 'cloud_nodes']
+    preview = data
+    rest = data
+    while rest != 0:
+        rest = valueToTest % pow(2, exponent)
```
Comment (giovtorres) [inline pyslurm/pyslurm.pyx]: `valueToTest` is undeclared

**Comment by giovtorres** [comment]
Could you have a look at the comments I left in the PR?  Thanks.

**Comment by yencli** [comment]
Ok, I removed the typo.

**Pull request merged**